deploy/charts/emqx/values.yaml

```diff
@@ -42,7 +42,7 @@ initContainers: {}
 
 ## EMQX configuration item, see the documentation (https://github.com/emqx/emqx-docker#emq-x-configuration)
 emqxConfig:
-  EMQX_CLUSTER__K8S__APISERVER: "https://kubernetes.default.svc:443"
+  EMQX_CLUSTER__K8S__APISERVER: \"https://kubernetes.default.svc:443\"
```

Comment:
this is a bit confusing to users. maybe we can improve this by adding scripts to `./docker-entrypoint.sh` and let it handle quotation stuff

Reply:
agree.